doc/rst/language/spec/modules.rst

```diff
@@ -260,63 +260,78 @@ nested modules.
+visibility of each individual symbol (:ref:`Visibility_Of_Symbols`). This can be
+done via the use statement (:ref:`Using_Modules`), the import
+statement (:ref:`Importing_Modules`) or qualified
+naming (:ref:`Explicit_Naming`).
 
 .. _Visibility_Of_A_Module:
 
 Visibility Of A Module
 ~~~~~~~~~~~~~~~~~~~~~~
 
-A top-level module is available for use (:ref:`Using_Modules`)
-anywhere. The visibility of a nested module is subject to the rules
-of :ref:`Visibility_Of_Symbols`, where the nested module is
+A top-level module is available for use (:ref:`Using_Modules`) or
+import (:ref:`Importing_Modules`) anywhere. The visibility of a nested module is
+subject to the rules of :ref:`Visibility_Of_Symbols`, where the nested module is
 considered a "module-scope symbol" of its outer module.
 
 .. _Visibility_Of_Symbols:
 
 Visibility Of A Module’s Symbols
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-A symbol defined at module scope is *visible* from outside the module
-when the ``privacy-specifier`` of its definition is ``public`` or is
-omitted (i.e. by default). When a module-scope symbol is declared
-``private``, it is not visible outside of that module. A symbol’s
-visibility inside its module is controlled by normal lexical scoping and
-is not affected by its ``privacy-specifier``. When a module’s symbol is
-visible (:ref:`Visibility_Of_A_Module`), the visible symbols
-it contains are accessible via the use
-statement (:ref:`Using_Modules`) or qualified
+A symbol defined at module scope is *visible* from outside the module when the
+``privacy-specifier`` of its definition is ``public`` or is omitted (i.e. by
+default). When a module-scope symbol is declared ``private``, it is not visible
+outside of that module. A symbol’s visibility inside its module is controlled by
+normal lexical scoping and is not affected by its ``privacy-specifier``. When a
+module’s symbol is visible (:ref:`Visibility_Of_A_Module`), the visible symbols
+it contains are accessible via the use statement (:ref:`Using_Modules`), import
+statement (:ref:`Importing_Modules`), or qualified
 naming (:ref:`Explicit_Naming`).
 
 .. _Using_Modules:
 
 Using Modules
 ~~~~~~~~~~~~~
 
-The ``use`` statement provides the primary means of accessing a module’s
-symbols from outside of the module. Use statements make both the
-module’s name and its public symbols available for reference within a
-given scope. For top-level modules, a use statement is required before
-referring to the module’s name or the symbols it contains within a given
-lexical scope.
+The ``use`` statement provides one of two primary ways to access a module’s
+symbols from outside of the module, the other being the ``import`` statement.
+Use statements make both the module’s name and its public symbols available for
+reference within a given scope. For top-level modules, a use or import statement
+is required before referring to the module’s name or the symbols it contains
+within a given lexical scope.
```

Comment:
Reading a bit more of this part of the spec, it may be too much styling. I'd still lean towards making them styled differently, but not thinking as strongly as I was before.

Reply:
I've added a few more here, but not everywhere